src/components/home/Footer.jsx

```diff
@@ -1,40 +1,45 @@
-import React from 'react';
+import React from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import {
-    faFacebook,
-    faTwitter,
-    faGoogle,
-    faGithub
+  faFacebook,
+  faTwitter,
+  faGoogle,
+  faGithub,
 } from "@fortawesome/free-brands-svg-icons";
-import './style.scss';
+import "./style.scss";
 
 const Footer = () => {
-    return <div>
-        <hr />
-        <footer>
-            <div className='icons'>
-                <a href="https://www.twitter.com/" className="twitter social">
-                    <FontAwesomeIcon icon={faTwitter} size="2x" />
-                </a>
-                <a href="https://www.gmail.com/" className="Gmail social">
-                    <FontAwesomeIcon icon={faGoogle} size="2x" /> </a>
-                <a href="https://www.facebook.com/" className="facebook social">
-                    <FontAwesomeIcon icon={faFacebook} size="2x" /></a>
-                <a href="https://www.github.com/" className="github social">
-                    <FontAwesomeIcon icon={faGithub} size="2x" /></a>
-            </div>
-            <ul className='footerNav'>
-                <li>  Home </li>
+  return (
+    <div>
+      <hr />
+      <footer>
+        <div className="icons">
+          <a href="https://www.twitter.com/" className="twitter social">
+            <FontAwesomeIcon icon={faTwitter} size="2x" />
+          </a>
+          <a href="https://www.gmail.com/" className="Gmail social">
+            <FontAwesomeIcon icon={faGoogle} size="2x" />{" "}
+          </a>
+          <a href="https://www.facebook.com/" className="facebook social">
+            <FontAwesomeIcon icon={faFacebook} size="2x" />
+          </a>
+          <a href="https://www.github.com/" className="github social">
+            <FontAwesomeIcon icon={faGithub} size="2x" />
+          </a>
+        </div>
+        <ul className="footerNav">
+          <li> Home </li>
 
-                <li> Products </li>
+          <li> Products </li>
 
-                <li> Blog </li>
+          <li> Blog </li>
 
-                <li> Conatct </li>
-            </ul>
-            <span dangerouslySetInnerHTML={{ "__html": "&copy; 2020" }} />
-        </footer>
+          <li> Conatct </li>
+        </ul>
+        <span dangerouslySetInnerHTML={{ __html: "&copy; 2020" }} />
```

Comment:
hm, why is there dangerously set inner html here? you should just be able to use a normal span

Reply:
but when I deleted the dangerouslyinnerhtml it gave me an error?

Reply:
please when you say there is an error include the error (not just this time but always)

Reply:
<span {{"&copy; 2020"}} /> so when it became like this 
i got two errors 
1-'...' expetced [39-16]
2-':' expected [39-30]

Reply:
or <span {{ __html: "&copy; 2020" }} />
i got only the first error

Reply:
just make it a normal react thing, you don't need this `__html`. treat the c2020 as any other text you would normally add to the page

Reply:
fixed